packages/components/src/MultiSelect/MultiSelect.tsx

```diff
@@ -26,13 +31,15 @@ export const MultiSelect = ({
   label,
   items,
   selectedValues,
+  description,
   onSelectedValuesChange,
   isOpen,
   onOpenChange,
   classNameOverride,
   ...restProps
 }: MultiSelectProps): JSX.Element => {
   const id = propsId ?? useId()
+  const descriptionId = useId()
```

Comment:
Might not be an issue, but do you think we should do the thing where if the consumer passes in an `id`, the description id is based on it? Eg. `{id}--description`?

Reply:
I've switched it for consistency